components/BottomNavigation/src/MDCBottomNavigationBarController.m

```diff
@@ -335,9 +347,56 @@ - (void)addContentViewController:(UIViewController *)viewController {
   if (viewController && doesNotContainViewController) {
     [self addChildViewController:viewController];
     [self.content addSubview:viewController.view];
-    [self addConstraintsForContentView:viewController.view];
+    [self addConstraintsForChildViewControllerView:viewController.view];
     [viewController didMoveToParentViewController:self];
   }
+  [self updateNavigationBarInsets];
+}
+
+- (void)viewDidLayoutSubviews {
+  [super viewDidLayoutSubviews];
+
+  [self updateNavigationBarInsets];
+}
+
+- (void)updateNavigationBarInsets {
+  UIEdgeInsets currentSafeAreaInsets = UIEdgeInsetsZero;
```

Comment:
could you add a description to this method on what it is trying to achieve? it is quite hard to infer just by reading the code

Reply:
Done. Please let me know if it looks good or needs greater elaboration.

Reply:
After our offline chat, I added more.